### ISSUE TITLE
Usersnap Feedback - 'Approach' is missing an option for conflict mediation

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#892]
**Sender**: scottowenfletcher@gmail.com
**Comment**: 'Approach' is missing an option for conflict mediation
[Open #587 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/587)

<a href='https://usersnappublic.s3.amazonaws.com/2019-10-09/02-42/db9148d1-478c-4f90-9850-051f78c65363.png'>![Usersnap Feedback - 'Approach' is missing an option for conflict mediation](https://usersnappublic.s3.amazonaws.com/2019-10-09/02-42/2c82bc14-1dde-4454-8989-d15644adc944.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-10-09/02-42/db9148d1-478c-4f90-9850-051f78c65363.png'>Download original image</a>

**Browser**: Chrome 77 (Windows 10)
**Referer**: [https://participedia.net/case/6135/edit?full=1](https://participedia.net/case/6135/edit?full=1)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 873

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by plscully** [comment]
@scottofletcher Isn't this something that we need to add to the list of things for review of the forms, which won't take place until many months from now? @ascott and @jesicarson  can weigh in on this, but it seems as though one by one suggestions for adding new values to the forms could be an  unnecessary distraction for D&T

**Comment by scottofletcher** [comment]
@plscully I'm not sure what you mean - I usersnap things for my own reference as well and add the editing label. Alanna also created an [editors repository](https://github.com/participedia/editors) in GitHub for the editing team to track issues

**Comment by plscully** [comment]
Oh, OK, I guess I still don't know exactly how this system works. I saw the note than Alanna had "self-assigned" this comment, but perhaps this is all automated and these aren't a distraction for her or Jesi.

**Comment by jesicarson** [comment]
Yep alanna automatically is assigned every usersnap. We glance at all of them and only tackle the ones that are related to our priorities or that are clearly bugs. Its ok for scott to use snaps to create editors issues.

**Comment by plscully** [comment]
Got it! Thank you, all, for doing such an amazing job of keeping on top of so many different types of details!

**Comment by scottofletcher** [comment]
ok but @plscully makes a good point - I should be more on top of assigning and labelling usersnaps I make for myself/editors so I don't distract D&T. it's too bad there's no way to move GitHub issues from one repository to another...

**Comment by ascott** [comment]
fwiw, i think it's ok to use the `usersnaps` repo as a triage area for all the site issues, including editor issues. from there, you may want to label it and just keep it in that repo, or we may want to create a new issue over on the `api` or `editors` repo to add more notes or for longer term priorities, etc. In which case you delete the one in the usersnaps repo. Does that process sound good to all of you? Other ideas?

it's correct that all usersnap issues are assigned to me, but it's not distracting for me.

**Comment by scottofletcher** [comment]
yes that sounds like a good idea - I've tended to open a new issue in the editor's repo and reference the usersnap when it's an ongoing issue or a long term project (see [here ](https://github.com/participedia/editors/issues/3) for example). would it be better for me to move the usersnap text over and delete it? I'd still like to keep small changes and reminders like this one as usersnaps w/the editors label (saves me having to open a new issues in the editors repo)

**Comment by jesicarson** [comment]
@scottofletcher yes you can use the usersnaps repo with editors label for small issues to avoid making new issues each time.

**Comment by scottofletcher** [comment]
ok sounds good gang! I'll make sure to assign myself and update the label right after I post something for editors
